src/senaite/core/schema/durationfield.txt

```diff
@@ -0,0 +1,54 @@
+Duration field
+==============
+
+The phone field stores timedeltas
```

Comment:
The duration field ...

Reply:
oops, fixed with https://github.com/senaite/senaite.core/pull/2482/commits/1b6e3510036be7a3927f0af8a4a3378aa20efc32